project_proposal/milestones.md

```diff
@@ -0,0 +1,43 @@
+# Milestones
+Milestones: What are the key milestones throughout the project? 
+
```

Comment:
Remove all meetings.

Reply:
Maybe that is fixed? can you take a look?

Reply:
I just changed it so it is not meeting any more but scrum planning and scrum meeting?

Reply:
That's fine yeah